kategory/src/test/kotlin/kategory/laws/MonoidKLaws.kt

```diff
@@ -0,0 +1,21 @@
+package kategory
+
+import io.kotlintest.properties.forAll
+
+object MonoidKLaws {
+
+    inline fun <reified F> laws(SGK: MonoidK<F>, AP: Applicative<F>, EQ: Eq<HK<F, Id.F>>, EQSemigroupK: Eq<HK<F, Int>>): List<Law> =
+            SemigroupKLaws.laws(SGK, AP, EQSemigroupK) + listOf(
+                    Law("MonoidK: Left identity", { monoidKLeftIdentity(SGK, EQ) }),
+                    Law("MonoidK: Right identity", { monoidKRightIdentity(SGK, EQ) }))
+
+    inline fun <reified F> monoidKLeftIdentity(SGK: MonoidK<F>, EQ: Eq<HK<F, Id.F>>): Unit =
+            forAll(genEmpty<F, Id.F>(SGK), { a ->
```

Comment:
Instead of explicitly using `Id.F` here can we have `F` be  generic arg to `MonoidKLaws` so callers can pass whatever `F` they wish?